CryptoPrices/Sources/MyCoin/Sources/MyCoin/CoinStatisticsSection/CoinStatisticsSection.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  CoinStatisticsSection.swift
+//  MyCoin
+//
+//  Created by Khanh on 03/01/2023.
+//
+
+import Styleguide
+import SwiftUI
+
+struct CoinStatisticsSection: View {
+
+    var body: some View {
+        VStack {
+            setUpCoinStatisticsItemView(
+                title: Strings.MyCoin.MarketCap.title,
+                price: coinDetailItem.marketCap,
+                isPriceUp: coinDetailItem.isMarketCapChangePercentage24HUp,
+                percentage: coinDetailItem.marketCapChangePercentage24H
+            )
+            setUpCoinStatisticsItemView(
+                title: Strings.MyCoin.Ath.title,
+                price: coinDetailItem.ath,
+                isPriceUp: coinDetailItem.isAthChangePercentageUp,
+                percentage: coinDetailItem.athChangePercentage
+            )
+            setUpCoinStatisticsItemView(
+                title: Strings.MyCoin.Atl.title,
+                price: coinDetailItem.atl,
+                isPriceUp: coinDetailItem.isAtlChangePercentageUp,
+                percentage: coinDetailItem.atlChangePercentage
+            )
```

Comment:
Just a matter of personal convention, feel free to ignore 🙏

The method `setUpCoinStatisticsItemView` creates and returns a (composite) view based on some inputs. (not too relevant but it's actually a pure function, that returns the same result with the same inputs when running _multiple times_ without side effects). So its name should be `makeCoinStatisticsItemView` or `createCoinStatisticsItemView` to better convey that meaning 🙏

To me, it's even better if we name it as simple as `coinStatisticsItemView` because it's part of another view.

The word `setUp` seems like a thing we mostly do _once_ with a sense that we prepare something or make something to be ready for other things to happen (and as a result usually has some side effects)

Reply:
Really thank you for the suggestion :D. The `coinStatisticsItemView` sounds more `swiftUI` to me.
I fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/84/commits/83786ad9d575b07d0f8de4b6c7c0030af3b78667.